metrics_utility/automation_controller_billing/report/base.py

```diff
@@ -41,7 +41,15 @@ def convert_cell(self, cell):
             return json.dumps(new_cell)
         # Otherwise, return the cell unchanged.
         return cell
+    def _fix_event_host_sample(self, mapping_dataframe, destination_dataframe):
+        if destination_dataframe is None:
+            return None
 
+        def concatenate_columns_mapping(row):
+            return f"{row['original_host_name']}__{str(row['install_uuid'])}__{str(row['job_remote_id'])}"
+
+        def concatenate_columns_destination(row):
+            return f"{row['host_name']}__{str(row['install_uuid'])}__{str(row['job_remote_id'])}"
```

Comment:
This will all be deleted pre-merge.  I added it just to get the coverage report to run.